.github/workflows/keep-build-cache-small.yaml

```diff
@@ -0,0 +1,119 @@
+name: keep-build-cache-small
+
+on:
+  workflow_call:
+  schedule:
+    - cron: 0 6 * * *
+
+env:
+  CACHE_THRESHOLD_GB: 6.6
+  IMAGE_NAME: autoware-buildcache
+  IMAGE_TAG: amd64-main
```

Comment:
Doesn’t the `arm64-main` cache also become a problem?

Reply:
Solved with: https://github.com/autowarefoundation/autoware/pull/5873/commits/5c4c8f667a7a9016038d11005bc5285bffae7b4d

https://github.com/autowarefoundation/autoware/actions/runs/13856227405

![image](https://github.com/user-attachments/assets/88dd50ac-d352-4ca5-bcb9-7231bfe28aee)